extensions/sql-migration/src/dialog/tableMigrationSelection/tableMigrationSelectionDialog.ts

```diff
@@ -295,6 +298,9 @@ export class TableMigrationSelectionDialog {
 			content: flex,
 			id: 'missingTablesTab',
 			title: constants.MISSING_TARGET_TABLES_COUNT(this._missingTableCount),
+			dispose: () => {
+				flex.dispose();
+			}
```

Comment:
is this change compatible with the current release of ADS in the stable gallery, or will the extension need to wait for the next release with related change in ADS before this change in the extension can be published in the stable as well.

Reply:
This would require a new ADS stable release for extensions to be able to call "dispose()" API on model-view components. You can however continue to build/publish an insiders-only extension to test the functionality with insiders build while in development.

With next ADS stable release, updated extension should be released to be compatible with new version onwards.

Reply:
@cheenamalhotra I'd suggest rolling this back for extensions like this that are in active development. We don't want to cause issues if they need to do a stable release between now and the next release

Reply:
Addressed.